src/tools/linker/external/corert/src/System.Private.CoreLib/shared/System/Reflection/RuntimeAssemblyName.cs

```diff
@@ -1,123 +1,125 @@
 // Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Could you unify these files with copies under `src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/` ?

Reply:
cc @jkotas

Reply:
I gave a shot to this very briefly and the NativeAOT files start to add more dependencies very quickly, so I will leave it as a separate item that can be investigated after the first commit (see https://github.com/dotnet/runtime/issues/77868).